query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_14696.rs

```diff
@@ -0,0 +1,75 @@
+use query_engine_tests::*;
+
+#[test_suite(schema(schema))]
+mod prisma_14696 {
+    fn schema() -> String {
+        include_str!("./prisma_14696.prisma").to_string()
```

Comment:
I like this for big schemas but I don't think it's an established pattern in this test suite. I'm happy to change it.